pages/linux/smbpasswd.md

```diff
@@ -0,0 +1,19 @@
+# smbpasswd
+
+> Change a user's SMB password.
```

Comment:
Perhaps we could add a new line here with a really common 'gotcha'?
```suggestion
> Change a user's SMB password.
> Samba users must also have a local Unix account.
```

I ran into this issue myself and spent _ages_ looking it up.

Reply:
You're right, this is important!